app/src/main/java/org/oppia/app/topic/lessons/StorySummaryAdapter.kt

```diff
@@ -94,34 +96,47 @@ class StorySummaryAdapter(
       binding.isListExpanded = isChapterListVisible
       binding.viewModel = storySummaryViewModel
 
-      val chapterSummaries = storySummaryViewModel.storySummary.chapterList
+      val chapterSummaries = storySummaryViewModel
+        .storySummary.chapterList
```

Comment:
Why are these lines being changed? Was this done automatically by ktlint?

Reply:
No, it is not done by ktlint. I purposely did this as it seems going out of grey line, although it is just because of the suggestion of variable but to keep a consistency that everything is inside the line, and that also not confuse other developers to use max line disable or not. 

`val chapterSummaries`**`:(Mutable)List<ChapterSummary!>!`**` = `